src/Auth0RestClient.js

```diff
@@ -61,6 +61,10 @@ Auth0RestClient.prototype.patch = function(/* [params], [callback] */) {
   return this.wrappedProvider('patch', arguments);
 };
 
+Auth0RestClient.prototype.post = function(/* [params], [callback] */) {
+  return this.wrappedProvider('post', arguments);
```

Comment:
we don't need 'post'. you have to use 'create', which is already being used through out the code base.

Reply:
0a6af3f